config/env/development.js

```diff
@@ -13,6 +13,9 @@ module.exports = {
     reference: true,
     i18n: false
   },
+  // in dev we have webpack-dev-server on 3000, and the real server on 3001
+  // @TODO NS consider the docker case
+  port: 3001,
```

Comment:
You could have this as `parseInt(process.env.FOOBAR || 3001, 10)` so that it's easy to just pass an environment variable to Docker container if it needs to be changed?

Reply:
I thought about using env variables, but given the configuration is already done via the files, I didn't want to add an additional mechanism - actually that TODO is out of date as I didn't need to change anything for the docker case...